### ISSUE TITLE
Compose UI: Show media toolbar by default

### ISSUE THREAD

**Issue by trwnh** [pixelfed/pixelfed#1367]
Currently it is a problem that image descriptions are not visible by default. Pixelfed should rethink the idea of having a "media toolbar" be separate and hidden. At the very least, the description field should be exposed by default just like the caption field.

**Comment by dansup** [comment]
Added in e4d6b002
